src/settings.js

```diff
@@ -0,0 +1,567 @@
+"use strict";
+(function () {
+    var fluid = window.fluid,
+        gpii = fluid.registerNamespace("gpii"),
+        ipcRenderer = require("electron").ipcRenderer;
+
+    fluid.registerNamespace("gpii.app.settings");
```

Comment:
I recommend a namespace of something like "gpii.pcp" - whilst this code is currently nested within the app project it probably won't always be so.